week8.tex

```diff
@@ -48,6 +53,7 @@ \subsection{Syntax}
 
 The syntax of types in CBPV is given by the following grammar:
 
+\todo{Use $\freeTy{A}$ and $\thunkTy{X}$ instead? This notation wasn't presented in class so it might be confusing}
```

Comment:
Added a few todos in the vanilla section for clarifications

Reply:
I think we should use it for consistency reasons

Reply:
Just to clarify, use the non-starred version or starred version? I think the non-starred would be more consistent?

Reply:
I get it, and I agree, but in theory we are working with the core language, not the surface syntax.... Or maybe we can consult Bob. I am okay if the entire document is replaced with non-starred, but you'll have to take care of what I've written

Reply:
And the starred version is the core language. I used all stars because I think it's more appropriate philosophy-wise, but your point is legit too

@HarrisonGrodin help 🥲

Reply:
I think the starred versions are the concrete syntax and the non-starred ones are the ABT, I mainly pick based on how nice they look

Reply:
I think it would probably be best to use `F` and `U` rather than the arrows, since that's the notation from class.

Reply:
Hmm, perhaps... totally up to you on that one.

Reply:
Well, I want consistency (either always star or no star) and you want F and U so we should get rid of all stars

Reply:
The main issue with no star is all the typing annotations get really cluttered and distracting, it's just unfortunate that the no star versions for F and U are what we want

Reply:
`\renewcommand` to override Bob's notation 😈

Reply:
Honestly not a bad idea, should we do that?

Reply:
Let's do it!